setup.cfg

```diff
@@ -26,3 +26,11 @@ packages =
 [entry_points]
 pytest11 =
     check-links = pytest_check_links.plugin
+console_scripts =
+    pytest_check_links = pytest_check_links.__main__:main
```

Comment:
The full name makes sense to me. I imagine there's a good chance of there already being a check-links tool somewhere.